package.json

```diff
@@ -9,9 +9,10 @@
 	"types": "./lib/index.d.ts",
 	"dependencies": {
 		"@types/node": "^14.14.32",
+		"@types/regression": "^2.0.2",
 		"@types/uuid": "^8.3.0",
-		"ardrive-core-js": "0.5.0",
-		"arweave": "1.10.11",
+		"ardrive-core-js": "0.5.1",
+		"arweave": "^1.10.16",
```

Comment:
Was this intended? Is arweave using semver?
We moved from a fixed version to any update but a major version...

Reply:
It was intended. The issue I was running into was a package conflict between  the `ardrive-core-js` dependency of `arweave` and the local CLI dependency of `arweave`.

To make sure they didn't conflict again, I specifically copied the version from the core commit below. Core was previously already using the `^` version syntax:

```diff
-		"arweave": "^1.10.11",
+		"arweave": "^1.10.16",
```

 https://github.com/ardriveapp/ardrive-core-js/pull/73/commits/c2b4faa638e7da0aa5fb35eac391f03135769639#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519